SYCL/ESIMD/api/functional/ctors/ctor_default.cpp

```diff
@@ -42,7 +42,7 @@ struct initializer {
 
 // Descriptor class for the case of calling constructor in variable declaration
 // context
-struct var_declaration {
+struct var_dec {
```

Comment:
```suggestion
struct var_decl {
```
The `dec` usually stands for the `decimal`

Reply:
I've reverted this comment [here](https://github.com/intel/llvm-test-suite/pull/588/commits/f4c3940c7788727dae45410460dde6655274266d#diff-e20a89556dfa9ec98f8eec4bea698de3975b780ff24a3bde1ced890af1d16581L46-L47)